scripts/update_sha.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+import argparse
+import yaml
+
+yaml_comments = """  # Make sure to always increase the revision when doing alpha/beta/rc releases!
+  # Example:
+  #
+  #   - 2.1.0-beta.1  => version=2.1.0, revision="1.beta.1"
+  #   - 2.1.0-beta.2  => version=2.1.0, revision="2.beta.2"
+  #   - 2.1.0-rc.1    => version=2.1.0, revision="3.rc.1"
+  #   - 2.1.0         => version=2.1.0, revision="4"
+  #   - 2.2.0-alpha.1 => version=2.2.0, revision="1.alpha.1"
+  #
+  # Only reset the revision once the version is bumped."""
+
+parser = argparse.ArgumentParser(description='Update package sha256 values in data.yml')
+parser.add_argument('--path', dest='yaml_path', help='The path to the yaml file to update.', required=True)
+parser.add_argument('--package-name', dest='package_name', help='The name of the package that should be updated.', required=True)
+parser.add_argument('--sha256', dest='checksum', help="The sha256 value that should be set for the package.", required=True)
+
+args = parser.parse_args()
+
+with open(args.yaml_path) as f:
+    data = yaml.load(f, Loader=yaml.FullLoader)
+    data[args.package_name]['sha256'] = args.checksum
```

Comment:
This is only updating the checksum but that's not enough, we also need to bump the versions and sometimes the suffix as well. What's the plan for that? :slightly_smiling_face:

Reply:
Yeah, I thought about that. My first thought was that copying the SHA's is the annoying part. Even if I still have to manually update the versions afterwards, it's still iteratively better. It does put the config into an unworking state, though, and I'd have to manually fix it then afterwards. But still less work than now.

On the other hand, adding parameters for the versions is pretty trivial. I'm just not sure how I want to get that data yet.

Reply:
@bernd Alright, I added arguments for the other values we update. That should be good for this side of the process, at least. It makes it updatable via script.

Now it's just a matter of passing it the right args.

Reply:
Hang on. I forgot... there's no way to set something to a blank value. :\